modules/apps/web-experience/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/display/context/SiteAdminDisplayContext.java

```diff
@@ -58,24 +69,51 @@
 import javax.portlet.PortletURL;
 
 import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
 /**
  * @author Pavel Savinov
+ * @author Alessio Antonio Rendina
  */
 public class SiteAdminDisplayContext {
 
 	public SiteAdminDisplayContext(
-			HttpServletRequest request,
+			HttpServletRequest request, HttpServletResponse response,
```

Comment:
Looking at all the changes, I think it would be better to create a separate DisplayContext for the step wizard. I will double check with the team though.